db/fault_injection_test.cc

```diff
@@ -228,21 +228,15 @@ class FaultInjectionTest : public testing::Test,
     return Status::OK();
   }
 
-#ifdef ROCKSDB_UBSAN_RUN
```

Comment:
Why these are being removed?

Reply:
After the change, UBSAN won't show warning, so no need to suppress warning any more.

Reply:
The git blame shows that the motivation for these were beyond the key function: https://github.com/facebook/rocksdb/pull/1577
Can you confirm that ubsan (both on bulk and using Makefile) would not complain after removing these lines?

Reply:
These attributes suppress warnings in the functions below it, which is Key(). Other functions still have things suppressed.